Jenkinsfile

```diff
@@ -101,7 +101,7 @@ pipeline {
                 sh 'build/run make -j\$(nproc) mod.check'
                 sh 'git diff-index --quiet HEAD || { echo "CHANGES FOUND. You may need to run make clean"; git status -s; exit 1; }'
                 // run the build
-                sh 'build/run make -j\$(nproc) build.all'
+                sh (script: "build/run make -j\$(nproc) IMAGES=${env.testProvider} build", returnStdout: true)
```

Comment:
If it's not a PR we still need to run `build.all`. A big part of the savings we are seeing with this change is skipping the arm64 build, but master and release builds can't skip that.

Reply:
`env.isOfficialBuild` should be useful to differentiate PR builds from master/release.